frontend/src/js/store/organizationSlice/types.ts

```diff
@@ -95,7 +95,7 @@ export interface Addon {
 }
 
 export interface BillingProfile {
```

Comment:
seeing this, it looks like `BillingProfileUpdate` would be the fitting type from the API, while the `BillingProfile` type there requires the address ☹️

Reply:
I think `BillingProfileUpdate` meant to be used for editing billing profile, while `BillingProfile` was supposed to come from the backend. The reality is quite different anyway :smile: